tests/integration/test_logcollector/test_log_filter_options/data/test_cases/cases_restrict_ignore_regex_values.yaml

```diff
@@ -0,0 +1,317 @@
+    ignore_regex: \\.ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log matches restrict"
+    matches: restrict
+
+- name: Test Restrict+Ignore  - pcre2+osregex engine - Matches Ignore Only
+  description: Test Restrict + Ignore tags (pcre2+osregex). Log matches ignore
+  configuration_parameters:
+    RESTRICT_REGEX: .*restrict
+    IGNORE_REGEX: \.ignore
+    RESTRICT_TYPE: pcre2
+    IGNORE_TYPE: osregex
+  metadata:
+    restrict_regex: .*restrict
+    ignore_regex: \\.ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log matches ignore"
+    matches: ignore
+
+- name: Test Restrict+Ignore - pcre2+osregex engine - Matches Both
+  description: Test Restrict + Ignore tags (pcre2+osregex). Log matches both
+  configuration_parameters:
+    RESTRICT_REGEX: .*restrict
+    IGNORE_REGEX: \.ignore
+    RESTRICT_TYPE: pcre2
+    IGNORE_TYPE: osregex
+  metadata:
+    restrict_regex: .*restrict
+    ignore_regex: \\.ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log matches ignore restrict"
+    matches: ignore - restrict
+
+- name: Test Restrict+Ignore - pcre2+osregex engine - Matches None
+  description: Test Restrict + Ignore tags (pcre2+osregex). Log matches None
+  configuration_parameters:
+    RESTRICT_REGEX: .*restrict
+    IGNORE_REGEX: \.ignore
+    RESTRICT_TYPE: pcre2
+    IGNORE_TYPE: osregex
+  metadata:
+    restrict_regex: .*restrict
+    ignore_regex: \\.ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log does not match"
+    matches: None
+
+# Test pcre2 + osmatch
+- name: Test Restrict+Ignore - pcre2+osmatch engine - Matches Restrict
+  description: Test Restrict + Ignore tags (pcre2+osmatch). Log matches restrict
+  configuration_parameters:
+    RESTRICT_REGEX: .*restrict
+    IGNORE_REGEX: ignore
+    RESTRICT_TYPE: pcre2
+    IGNORE_TYPE: osmatch
+  metadata:
+    restrict_regex: .*restrict
+    ignore_regex: ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log matches restrict"
+    matches: restrict
+
+- name: Test Restrict+Ignore - pcre2+osmatch engine - Matches Ignore Only
+  description: Test Restrict + Ignore tags (pcre2+osmatch). Log matches ignore
+  configuration_parameters:
+    RESTRICT_REGEX: .*restrict
+    IGNORE_REGEX: ignore
+    RESTRICT_TYPE: pcre2
+    IGNORE_TYPE: osmatch
+  metadata:
+    restrict_regex: .*restrict
+    ignore_regex: ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log matches ignore"
+    matches: ignore
+
+- name: Test Restrict+Ignore - pcre2+osmatch engine - Matches Both
+  description: Test Restrict + Ignore tags (pcre2+osmatch). Log matches both
+  configuration_parameters:
+    RESTRICT_REGEX: .*restrict
+    IGNORE_REGEX: ignore
+    RESTRICT_TYPE: pcre2
+    IGNORE_TYPE: osmatch
+  metadata:
+    restrict_regex: .*restrict
+    ignore_regex: ignore
+    log_sample: "Nov 10 12:19:04 localhost sshd: log matches ignore restrict"
+    matches: ignore - restrict
+
+- name: Test Restrict+Ignore - Matches None
```

Comment:
```suggestion
- name: No log match - Ignore and restrict tags with pcre2 and osmatch regex
```

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/3582/commits/b7da06b96bffcb3001f1d433cfce78e878f00905